<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="EN" lang="EN" dir="ltr"><!-- InstanceBegin template="/Templates/bio-inside-temp.dwt" codeOutsideHTMLIsLocked="false" -->
<head>
<!-- InstanceBeginEditable name="doctitle" -->
<title>Department of Biology and Physics - Research - Biotechnology (BTEC) and Molecular, Cell, Developmental Biology (MCDB)</title>
<!-- InstanceEndEditable -->
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1" />
<meta http-equiv="imagetoolbar" content="no" />
<link href="../images/bkgd/favicon.ico" rel="shortcut icon" title="shortcut icon" />
<script type="text/javascript" src="../scripts/jquery-1.4.1.min.js"></script>
<script type="text/javascript" src="../scripts/jquery.slidepanel.setup.js"></script>
<script type="text/javascript" src="../scripts/jquery.cycle.min.js"></script>
<script type="text/javascript" src="../scripts/jquery.cycle.setup.js"></script>
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-22810300-5', 'kennesaw.edu');
  ga('send', 'pageview');

</script>
<style type="text/css">
a:link {
	color: #007B85;
	text-decoration: none;
}
a:visited {
	color: #007B85;
	text-decoration: none;
}
a:hover {
	color: #099;
	text-decoration: none;
}
a:active {
	color: #007B85;
	text-decoration: none;
}
h1 {
	font-size: 10px;
	color: #FFF;
}
body,td,th {
	font-size: 14px;
	font-family: Arial, Helvetica, sans-serif;
	font-style: normal;
	line-height: normal;
	font-weight: normal;
	font-variant: normal;
	text-decoration: none;
	text-transform: none;
	vertical-align: top;
	background-repeat: repeat-x;
	color: #333;
}
body {
	background-color: #000;
}
</style>
<!-- InstanceBeginEditable name="head" -->
<!-- InstanceEndEditable -->
<link href="../styles/featured_slide.css" rel="stylesheet" type="text/css" />
<link href="../styles/forms.css" rel="stylesheet" type="text/css" />
<link href="../styles/ie.css" rel="stylesheet" type="text/css" />
<link href="../styles/layout.css" rel="stylesheet" type="text/css" />
<link href="../styles/loginslider.css" rel="stylesheet" type="text/css" />
<link href="../styles/menu_style.css" rel="stylesheet" type="text/css" />
<link href="../styles/navi.css" rel="stylesheet" type="text/css" />
<link href="../styles/tables.css" rel="stylesheet" type="text/css" />
</head>
<body>
<div class="wrapper col0">
  <div id="topbar">
    <div id="slidepanel">
      <div class="topbox">
        <div align="center"><b><a href="http://web.kennesaw.edu/azindex" title="A-Z Index" class="style7">A-Z Index</a></b>&nbsp;|&nbsp;				
                  <a href="http://www.kennesaw.edu" title="KSU Website" class="style7"><strong>KSU Website</strong></a>
  &nbsp;</td>
                  </tr>
        </div>
        
                <tr>
					<td>
					<form method="GET" action="http://gsearch.kennesaw.edu/search">
					  <p align="right">
						<input type="text" name="q" size="22" style="border: 1px solid #C0C0C0; padding: 4px">
						<input type="submit" value="SEARCH" name="B1" style="color: #FFFFFF; font-family: Trebuchet MS; font-size: 10pt; border: 1px solid #F5BA12; padding-left: 4px; padding-right: 4px; padding-top: 0px; padding-bottom: 0px; background-color: #000000">

						<input type="hidden" name="site" value="default_collection">

   <input type="hidden" name="client" value="default_frontend">

   <input type="hidden" name="output" value="xml_no_dtd">

   <input type="hidden" name="proxystylesheet" value="default_frontend">

						</p>

					</form>
      </div>
      <br class="clear" />
    </div>
    <div id="loginpanel">
      <ul>
        <li class="left"></li>
        <li class="right" id="toggle"><a id="slideit" href="#slidepanel">SEARCH &raquo;</a><a id="closeit" style="display: none;" href="#slidepanel">Close Panel</a></li>
      </ul>
    </div>
    <br class="clear" />
  </div>
</div>
<!-- ####################################################################################################### -->
<div class="wrapper col1">
  <div id="header">
    <div id="logo">
      <h1 align="right"><a href="../index.html"><img src="../images/bkgd/header.png" alt="Department of Biology and Physics" title="Department of Biology and Physics" width="480" height="60" border="0" align="middle" /></a></h1>
</div>
    <div id="topnav">
      <ul>
        <li class="active"><a href="../about/index.html" title="About" target="_self">ABOUT</a></li>
        <li><a href="../academics/index.html" title="Academics" target="_self">ACADEMICS</a></li>
        <li><a href="../resources/index.html" title="Resources" target="_self">RESOURCES</a></li>
        <li><a href="../people/index.html" title="People" target="_self">PEOPLE</a></li>
        <li><a href="../happening/index.html" title="Happening" target="_self">HAPPENING</a></li>
      </ul>
    </div>
<br class="clear" />
  </div>
</div>
<img src="../images/bkgd/yellbar.png" alt="yellow bar graphic image" width="100%" height="18" />
<div class="wrapper col2">
  <!-- InstanceBeginEditable name="breadcrumb" -->
  <div id="breadcrumb">
    <ul>
      <li class="first"></li>
  <li><a href="../index.html" title="Department of Biology and Physics" target="_self">Home</a> &#187; <a href="../resources/index.html" title="Resources" target="_self">Resources</a> &#187; <a href="../research/index.html" title="Research" target="_self">Research</a> &#187; Molecular, Cell, Developmental Biology (MCDB)</li></ul>
  </div><!-- InstanceEndEditable -->
</div>
<div></div>
<!-- ####################################################################################################### -->
<div class="wrapper col3">
<div id="container"><!-- InstanceBeginEditable name="body" -->
  <div id="content">
    <div id="comments3">
      <h1>Molecular, Cell, Developmental Biology (MCDB)</h1>
      <p>This group of faculty conduct research in all areas of Biotechnology and the areas of Molecular, Cellular, Microbiology, Genetics, Physiology and Developmental Biology.</p>
      <h1><a name="jumptotop" id="jumptotop"></a>Faculty</h1>
      <ul>
        <li><strong><a href="#jump1" title="jump1">Premila Achar, Ph. D.</a></strong></li>
        <li><strong><a href="#jump2" title="jump2">Eric Albrecht, Ph. D.</a></strong></li>
        <li><strong><a href="#jump3" title="jump3">Estella Chen, Ph. D.</a></strong></li>
        <li><strong><a href="#jump4">Marcus C. Davis, Ph. D.</a></strong></li>
       <li><strong><a href="#jump5">Melanie Griffin, Ph. D.</a></strong></li>
        <li><strong><a href= "#jump6">Xueya Hauge, Ph. D.</a></strong></li>
        <li><strong><a href= "#jump7">Jerald D. Hendrix, Ph. D.</a></strong></li>
        <li><strong><a href="#jump8">Martin Hudson, Ph. D.</a></strong></li>
        <li><strong><a href="#jump9">Army Lester, Ph. D.</a></strong></li>
        <li><strong><a href="#jump10">Jean Lu, Ph. D.</a></strong></li>
        <li><strong><a href="#jump11">Thomas McElroy, Ph. D.</a></strong></li>
        <li><strong><a href="#jump12">Donald McGarey, Ph. D.</a></strong></li>
        <li><strong><a href="#jump13">Scott Nowak, Ph. D.</a></strong></li>
        <li><strong><a href="#jump14">John Salerno, Ph. D.</a></strong> </li>
        <li><strong><a href="#jump15">Susan M.E. Smith, Ph. D.</a></strong>        </li>
        <li><strong><a href="#jump16">Dale Lynn Vogelien, Ph. D.</a></strong></li>
        </ul>
      <h1>Featured Faculty Research</h1>
      <p><strong><img class="imgl" src="../images/125x125/pachar-research.jpg" alt="Photo of Dr. Premila Achar" title="Photo of Dr. Premila Achar" width="125" height="125" /><a name="jump1" id="jump1"></a>Premila Achar, Ph. D.</strong><br />
        <a href="../people/pachar.html" title="Dr. Premila Achar's contact information" target="_self">contact</a></p>
      <p><strong>Research Interests:</strong> Aflatoxins are carcinogenic to humans and other animals, and accumulation of aflatoxins in peanut kernels can influence marketing of peanut and peanut products. Breeding for varieties resistant to&nbsp;<em>A. flavus&nbsp;</em>has not been very promising, as evidenced by serious economic losses in the peanut market due to aflatoxin outbreaks year after year. Breeding resistant cultivars would be an effective approach to preventing aflatoxin accumulation in peanuts. However, traditional breeding is a costly, laborious and time consuming process. The goal of this research is to transform peanuts with glucanase and chitinase genes against&nbsp;<em>A. flavus</em>, the most common aflatoxin producing&nbsp;<em>Aspergillus&nbsp;</em>sps. found in peanuts and other food and agricultural products.</p>
      <p>Research also include expression of pathogenesis-related (PR) proteins such as chitinase and &beta;-1,3-glucanase and aflatoxin at cellular and subcellular levels in susceptible, tolerant, and transgenic peanut lines by immunocytochemical approaches. The intellectual meritof this research is to produce basic information about a little studied area, the characterization of PR proteins and disease resistance in transgenic peanuts. Basic molecular and biochemical studies in tolerant/transgenic peanut varieties will provide information on tolerance/resistance and on prevention of aflatoxin accumulation in peanuts. The in Situ localization of induced molecules in the presence of foreign genes from the cyto and immunocytochemical approaches will form the basis for future research in the development of biological control alternatives for&nbsp;<em>A. flavus&nbsp;</em>in peanuts. Broader impactsinclude the research&rsquo;s potential to make progress in developing an effective approach to eliminating aflatoxin accumulation in peanuts. Moreover, developing new peanut transgenic cultivars with resistance using genetic engineering will have great potential in breeding resistance to&nbsp;<em>A. flavus&nbsp;</em>infection, and the problem of aflatoxin management could be solved if peanut cultivars are resistant against&nbsp;<em>A.</em>&nbsp;<em>flavus&nbsp;</em>colonization. Also, the research goal is &nbsp;to train students in hands-on experiments in molecular, biochemical and immunocytochemical techniques and to engage them&nbsp; in a research lab environment.</p>
      <br />
      <p><strong><img class="imgr" src="../images/125x125/albrecht-research.jpg" alt="Photo of Dr. Premila Achar" title="Photo of Dr. Premila Achar" width="125" height="125" /><a name="jump2" id="jump2"></a>Eric Albrecht, Ph. D.<br /></strong>
        <a href="../people/ealbrecht.html" title="Dr. Eric Albrecht's contact information" target="_self">contact</a></p>
      <p><strong>Research interests:&nbsp;</strong>Our goal is to understand molecular mechanism(s) used by vascular tissue to resist substrate contact loss. Our model system utilizes crude hemorrhagic snake venom to induce cellular detachment of human umbilical vein endothelial cells. We are particularly interested in the function of metallothioneins during this process. The specific aims of our research include the following: 1) Identify venom protein(s) which induce MT gene activation, 2) Determine metallothioneins involvement in endothelial focal contact disassociation during hemorrhagic venom stimulation, 3) Determine signaling pathways mediated by metallothioneins after exposure to hemorrhagic venom.</p>
      <br />
      <p><strong><img class="imgl" src="../images/125x125/echen-research.jpg" alt="Photo of Dr. Premila Achar" title="Photo of Dr. Estella Chen" width="125" height="125" /><a name="jump3" id="jump3"></a>Estella Chen, Ph. D.</strong><br />
        <a href="../people/echen.html" title="Dr. Estella Chen's contact information" target="_self">contact</a></p>
      <p><strong>Research interests:&nbsp;</strong>In my lab we study the potential role of mitochondria in cancer formation. The majority of cancers&mdash;including cancers of the colon, breast, bladder, head and neck, lung, prostate and ovary&mdash;have abnormalities in their metabolism and exhibit somatic mutations in the mitochondrial chromosome (mtDNA). &nbsp; Cancer cells are largely glycolytic, deriving their cellular energy, ATP, from glycolysis rather than mitochondrial respiration. &nbsp;Cancer cells are also frequently defective in programmed cell death (apoptosis), which is a mechanism used by normal tissue to prevent the accrual of damaged cells. &nbsp;Normal mitochondria function is necessary for both aerobic ATP production and programmed cell ceath. Defects in mitochondria will also generate increased levels of reactive oxygen species via the electron transport chain, bringing in a mutagenic source for DNA damage in cancer genes such as proto oncogenes and tumor-suppressor genes. Thus, defects in mitochondria potentially promote cancer cell development in these three mechanisms.&nbsp;&nbsp; Over 65% of cancers examined show mutations in the mitochondrial chromosome (mtDNA), which codes for components of the electron transport chain. &nbsp;Do these mtDNA mutations play a causal role in cancer formation, or are they merely a side effect in the abnormal cancer cell microenvironment?<br />
        <br />
        To address this question, we characterize somatic (not inherited, acquired during the lifetime of the individual) mtDNA mutations in cancer. What research projects do you have currently?</p>
      <ul>
        <li>Bioinformatics project &#8212; alignment of interspecies mitochondrial a.a. sequences to assess significance of a.a. changes in cancer residues; project 2 &#8212; alignment of mitochondrial rRNA and tRNA sequences to assess significance of RNA changes in cancer associated mtDNA sequences. Being an author on a paper is a possibility for the student but not a guarantee.</li>
        <li>Lab project &#8212; use of RT PCR to determine the pattern of ROS (reactive-oxygen species) DNA damage in the mitochondrial chromosome</li>
        <li>Statistical analysis &#8212; examination of somatic mtDNA mutation from aged, pre-neoplastic tissue</li>
        </ul>
      <p><a href="#jumptotop">(Back to top) </a></p>
<br />
      <p><strong><img class="imgr" src="../images/125x125/mdavis-research.jpg" alt="Photo of Dr. Marcus Davis" title="Photo of Dr. Marcus Davis" width="125" height="125" /><a name="jump4" id="jump4"></a>Marcus C. Davis, Ph. D.</strong><br />
        <a href="../people/mdavis.html" title="Dr. Marcus C. Davis' contact information" target="_self">contact</a><br />
       
      <p><strong>Research interests:&nbsp;</strong>  The Davis Lab studies the developmental mechanisms of vertebrate morphogenesis, with a emphasis on musculoskeletal complexes such as paired appendages (fins and limbs) and jaws. We take a broadly comparative approach, utilizing basal actinopterygians such as the North American paddlefish Polyodon spathula, chondrichthyans, and even basal tetrapods to test hypotheses about the evolution of development. <a href="http://science.kennesaw.edu/~mdavi144/Research.html" title="Dr. Marcus Davis's Lab" target="_self"> Click for more information about his research</a><br />
        <a href="#jumptotop" title="back to top">(Back to top)</a></p>
      <p><strong><img class="imgl" src="../images/125x125/mgriffin-research.jpg" alt="Photo of Dr. Melanie Griffin" title="Photo of Dr. Melanie Griffin" width="125" height="125" /><a name="jump5" id="jump5"></a>Melanie Griffin, Ph. D.</strong><br />
        <a href="../people/mgriffin.html" title="Dr. Griffin's contact information" target="_self">contact</a><br />
      <p><strong>Research interests:&nbsp;</strong>  A leucine responsive protein (Lrp) has been demonstrated to play a role as a global regulator of cellular metabolism in response to leucine in the intestinal bacterium Escherichia coli.   In E. coli, Lrp binds DNA to activate or repress the gene expression for up to thirty operons and has likely evolved to suit its particular biological niche.  Some genes involved include amino acid biosynthesis and the adherence fimbriae structure.  This proposal speculates about the possible role of Lrp as a global regulator in another bacterium,Pseudomonas aeruginosa.  P. aeruginosa is found ubiquitously and harmlessly throughout the environment but can be an important opportunistic, human pathogen.  The most important aspect of P. aeruginosa disease is, once an internal infection is in place within its human host, there is no known cure or effective therapy. Pseudomonas is protected by two membranes surrounded by an extremely thick, mucoid structure, which helps it to be resistant to virtually all known antibiotic therapies and disinfectants.  It also produces adherence fimbriae (pili), which are critical for the initial infection.  Based on the E. coli profile, study of the Lrp protein may allow us to identify an important factor in the regulation of genes involved in cellular metabolism and possibly pathogenesis in P. aeruginosa.  Specifically, this study seeks to determine the number and diversity of genes acted upon by Lrp to determine if Lrp is a global or local regulator of gene function in P. aeruginosa. <br />
        <a href="#jumptotop" title="back to top">(Back to top)</a></p>
       <p><strong><img class="imgr" src="../images/125x125/xhauge-research.jpg" alt="Photo of Dr. Xueya Hauge" title="Photo of Dr. Xueya Hauge" width="125" height="125" /><a name="jump6" id="jump6"></a>Xueya Hauge, Ph. D.</strong><br />
        <a href="../people/xhauge.html" title="Dr. Hauge's contact information" target="_self">contact</a><br />
       <p><strong>Research interests:&nbsp;</strong> My research interest  is to study the molecular mechanism of human chromosomal diseases, including 9p  deletion syndrome. The terminal deletion of the short arm of human chromosome 9  (9p-) is associated with trigonocephaly, severe mental retardation, and  dysmorphic facial features. The gene(s) responsible for the 9p- syndrome remain  to be elucidated. To date, we have identified over 270 highly conserved  non-coding sequences between the human and mouse or between the human and  zebrafish using database analysis. These sequences are located within the  critical region for 9p- syndrome. Studies have shown that highly conserved  non-coding sequences may contain regulatory elements, which enhance (enhancers)  or suppress (suppressors) gene expression. We are in the process of testing  some of these 270 highly conserved non-coding sequences using a dual luciferase  assay. In this assay, the luciferase gene, isolated from the firefly, converts  luciferin to oxyluciferin while emitting fluorescent light. Therefore, the  higher the luciferase activity is the brighter the fluorescent light will be.  By inserting our putative regulatory sequences in front of the luciferase gene,  we are able to detect and measure an increase or a decrease of the luciferase  activity as compared to the activity of a second fluorescent gene, Renilla  gene, isolated from sea pansy. Polymerase chain reaction, gene cloning,  mammalian tissue culture, and gene transfection techniques are used, in  addition to the luciferase assay method. The ultimate goal for this approach is  to find regulate sequences for the most likely candidate genes responsible for  the 9p- syndrome.<br />
        <a href="#jumptotop" title="back to top">(Back to top)</a></p>
       <p><strong><img class="imgl" src="../images/125x125/jhendrix-research.jpg" alt="Photo of Dr. Jerald Hendrix" title="Photo of Dr. Jerald Hendrix" width="125" height="125" /><a name="jump7" id="jump7"></a>Jerald Hendrix, Ph. D.</strong><br />
        <a href="../people/jhendrix.html" title="Dr. Hendrix's contact information" target="_self">contact</a><br />
       <p><strong>Research interests:&nbsp;</strong> My research focuses  on mathematical modeling of bacterial growth in continuous culture. A  mathematical model for microbial growth is a set of differential equations that  predict changes in microbial concentration and nutrient concentration over  time.&nbsp; Models for microbial growth in  batch and continuous culture under limiting nutrient conditions were first  proposed and studied in the 1950s.&nbsp;  Subsequent work has focused on modifications of the original models to  account for various phenomena.&nbsp; In 2001,  I worked with Dr. Sean Ellermeyer in the Mathematics department at KSU to  develop and test a new mathematical model that accounted for a time in the  responses of bacterial growth to nutrient addition.&nbsp; My current research is focused on extending  this model to make predictions about the competition of bacteria in mixed  culture.<br />
        <a href="#jumptotop" title="back to top">(Back to top)</a></p>
      
      <p><strong><img class="imgr" src="../images/125x125/mhudson-research.jpg" alt="Photo of Dr. Martin Hudson" title="Photo of Dr. Martin Hudson" width="125" height="125" /><a name="jump8" id="jump8"></a>Martin Hudson, Ph. D.</strong><br />
        <a href="../people/mhudson.html" title="Dr. Martin Hudson's contact information" target="_self">contact</a><br />
      <p><strong>Research interests:&nbsp;</strong>   The Hudson lab uses nematode and stem cell models to investigate cell migration during nervous system development, with particular emphasis on the roles of heparan sulfate proteoglycans (HSPGs) and Eph receptor tyrosine kinases in this process. 
       <em>C. elegans</em> nematodes offer many advantages for investigating nervous system development. The animals are transparent, allowing the use of fluorescent labels to monitor cell movement and morphology. Also, well-developed genetic techniques and a fully sequenced genome facilitate investigation of gene function in this organism. We use time-lapse video microscopy and other techniques to understand how neuroblast migration contributes to<em> C. elegans</em> embryonic development. In addition, we are investigating how different HSPG core proteins contribute to axon branch formation.</p>
      <p>Stem cells offer great hope for personalized medicine and drug discovery. We have adopted a mouse stem cell-to-forebrain neural development model, originally developed by Austin Smith, to investigate cell migration during forebrain development. Mouse embryonic stem cells plated at low density undergo a default developmental program that recapitulates that seen during cortical layer specification of the mouse brain. We are using this model to identify genes required for cell migration during layer specification, with an emphasis on the role of EphR/ephrin function in neurogenesis. <a href="http://science.kennesaw.edu/~mhudso28/index.html" title="Dr. Martin Hudson's Lab" target="_self"> Click for more information about his research</a><br />
        <a href="#jumptotop" title="back to top">(Back to top) </a></p>
       
      <p><strong><img class="imgl" src="../images/125x125/alester-research.gif" alt="Photo of Dr. Army Lester" title="Photo of Dr. Army Lester" width="125" height="125" /><a name="jump9" id="jump9"></a>Army Lester, Ph. D.</strong><br />
        <a href="../people/alester.html" title="Dr. Lester's contact information" target="_self">contact</a><br />
      <p><strong>Research interests:&nbsp;</strong>  Army Lester&rsquo;s research interests are divided into two areas:  experimental embryology and improving K-16 STEM educational outcomes. The  embryology work focuses primarily on the growth and development of cells,  organs and whole chick embryos in vitro to determine how environmental factors  influence normal developmental and physiological patterns.&nbsp; One of Dr. Lester&rsquo;s projects involves  defining the conditions necessary to hatch shell-less chick embryos and allow  them to grow into normal adults. &nbsp;His  work also includes studies on embryo heart cultures, which may be used as an  investigative tool for stem cell research.&nbsp;  His work on STEM education focuses on the use of personalized  instructional materials that places the student as the central character in the  study of STEM principles. Common household products are used to explain complex  STEM principles thus allowing students to build on what they already know.&nbsp;&nbsp; His work also looks at 3-D mental modeling,  repetitions and multiple learning styles to foster long-term memory and  comprehension.<br />
        <a href="#jumptotop" title="back to top">(Back to top)</a></p>
      <p><strong><img class="imgr" src="../images/125x125/jlu-research.jpg" alt="Photo of Dr. Jean Lu" title="Photo of Dr. Jean Lu" width="125" height="125" /><a name="jump10" id="jump10"></a>Jean Lu, Ph. D.</strong><br />
        <a href="../people/jlu.html" title="Dr. Lu's contact information" target="_self">contact</a><br />
      <p><strong>Research interests:&nbsp;</strong>My research interests focus on the ecology and genetics of  bacterial viruses (bacteriophages) in industrial fermentations and the  application of bacteriophages to control bacterial pathogens. In addition, I am  also interested in the development of HIV/AIDS vaccines.<br />
        <a href="#jumptotop" title="back to top">(Back to top)</a></p>
      <p><strong><img class="imgl" src="../images/125x125/tmcelroy-research.jpg" alt="Photo of Dr. Thomas McElroy" title="Photo of Dr. Thomas McElroy" width="125" height="125" /><a name="jump11" id="jump11"></a>Thomas McElroy, Ph. D.</strong><br />
        <a href="../people/tmcelroy.html" title="Dr. McElroy's contact information" target="_self">contact</a><br />
      <p><strong>Research interests:&nbsp;</strong>My research interests  are in molecular ecology.&nbsp; In particular,  I am interested in collaborative and continuing research that will increase our  understanding of how ecological factors structure genetic variation within and  among natural populations and communities.&nbsp;  I use descriptive field work to generate hypotheses that can be tested  through field and laboratory experimentation.&nbsp;  I have created opportunities for undergraduate research at KSU by  collaborating with faculty both here and at other institutions with different  expertise from ecologists, physiologists, and microbiologists. My research  program offers projects that are local (Genetic Characterization of the Stone  Roller (Campostoma anomalum) in the Etowah River System), and I offer research  opportunities through study abroad (Mexico: Differences in Source Water Use of  Woody Vegetation of Dzibilchatun in the Yucatan Peninsula, and Canada:  Population dynamics of Wood Frogs (Rana sylvatica) in northern Canada).&nbsp; The field and laboratory nature of data  collection associated with my research allows for the inclusion of both highly  motivated directed study students, and student volunteers who may be unsure  about their specific interests in the area of molecular ecology and  biology.&nbsp; I follow a student&ndash;centered  learning model that provides hands-on experience with laboratory techniques  such as PCR and DNA sequencing. This experience allows students to appreciate  the unity and diversity present in biological systems and the challenges  biologists face when trying to ask and answer what seem to be relatively  straight-forward questions.<br />
        <a href="#jumptotop" title="back to top">(Back to top)</a></p>
      <p><strong><img class="imgr" src="../images/125x125/dmcgarey-research.jpg" alt="Photo of Dr. Don McGarey" title="Photo of Dr. Don McGarey" width="125" height="125" /><a name="jump12" id="jump12"></a>Donald McGarey, Ph. D.</strong><br />
        <a href="../people/dmcgarey.html" title="Dr. McGarey's contact information" target="_self">contact</a><br />
      <p><strong>Research interests:&nbsp;</strong>The primary focus of my work is in water-borne  diseases.<strong>&nbsp; </strong>Aeromonads  are widespread, aquatic bacteria of which some are pathogens of aquatic animals  and humans. <strong><em>Aeromonas hydrophila </em></strong>is  one species that<strong></strong>causes  various diseases in animals including highly destructive, soft tissue  infections. In humans, the disease can manifest as a localized cellulitis or a  disseminated &ldquo;flesh-eating&rdquo; condition. In fish, amphibians and reptiles the  disease can occur as localized lesions or a blood-borne septicemia. Current projects  in this area focus on the tissue-degrading enzymes produced by this bacterium,  development of a PCR diagnostic test to detect this bacterium in water and  clinical samples, the use of multi-locus sequence typing as a means of  identifying closely-related species and the development of a suitable animal  model to study bacterium: host dynamics. Additional work in my lab revolves  around water quality analysis. Water suitable for human consumption and  recreation, as well as ecosystem sustainability require testing methods that  can indicate when there are problems and where the problems may originate.&nbsp; Microbial Source Tracking (MST) is the use of  fecal-associated microbes, primarily bacteria, to determine the presence of  potential harmful microbes in water, the level of Public Health risk for those  using the water and the potential source of the waste. Current projects in this area focus on the use of specific bacteria as  indicators of pollution sources and the use of diagnostic tests to detect the  levels of these microbes.<br />
        <a href="#jumptotop" title="back to top">(Back to top)</a>      </p>
     <p><strong><img class="imgl" src="../images/125x125/snowak-research.png" alt="Photo of Dr. Scott Nowak" title="Photo of Dr. Scott Nowak" width="125" height="125" /><a name="jump13" id="jump13"></a>Scott Nowak, Ph. D.</strong><br />
        <a href="../people/snowak.html" title="Dr. Scott Nowak's contact information" target="_self">contact</a><br />
      <p><strong>Research interests:&nbsp;</strong>The Nowak lab uses myogenesis as a model system to study mechanisms of   gene regulation during organismal development. &nbsp;During muscle formation,   mononucleated myoblasts differentiate by expressing muscle-specific   gene products under the control of these transcription factors, and fuse   together to form a multinucleated muscle fiber. &nbsp;The differentiation   and fusion process is essential not only for embryonic muscle   development but also for the maintenance and regeneration of healthy   adult muscle in mammals. &nbsp;Understanding the differentiation and fusion   process of myogenesis is critical for improving efficacy of therapies   aimed at mitigating or reversing the devastating effects of muscle   wasting, which can occur due to muscle injury, stroke, cancer cachexia,   and muscular dystrophy disorders. &nbsp;We use a combination of Drosophila   genetics and mouse tissue culture systems to understand regulation of   the different events that occur during muscle formation.<br />
        <a href="#jumptotop" title="back to top">(Back to top)</a>      </p>
      <p><strong><img class="imgr" src="../images/125x125/jsalerno-research.jpg" alt="Photo of Dr. John Salerno" title="Photo of Dr. John Salerno" width="125" height="125" /><a name="jump14" id="jump14"></a>John Salerno, Ph. D.</strong><br />
        <a href="../people/jsalerno.html" title="Dr. John Salerno's contact information" target="_self">contact</a><br />
      <p><strong>Research interests:&nbsp;</strong>A major focus of  Salerno&rsquo;s lab is the study of catalysis and control in nitric oxide synthase  (NOS) enzymes using physical biochemical methods and molecular genetics. Nitric  Oxide is the primary controller of vascular tone (hence blood pressure) in  mammals. It is a major signal in insulin secretion, peristalsis, sexual  arousal, angiogenesis, cardiac, lung and airway function, and is a  neurotransmitter and a reporter of skeletal muscle contraction. Work includes  molecular modeling of NOS and other proteins, and simulation of biological  processes including catalysis and cooperative ligand binding in NOS and other  enzymes, and diffusion of signals from pulsed sources in various cellular  geometries.&nbsp; Related interests include  studies of other heme containing enzymes (e.g., peroxidases and NADPH oxidases)  and enzymes related to signaling.<br />
        <a href="#jumptotop" title="back to top">(Back to top)</a>      </p>
        <img src="../images/125x125/ssmith-research.jpg" alt="Photo of Dr. Susan M.E. Smith" width="125" height="125" class="imgl" title="Photo of Dr. Susan Smith" /><p><strong><a name="jump15" id="jump15"></a>Susan M. E. Smith, Ph.D.</strong><br />
        <a href="../people/ssmith.html" title="Dr. Susan Smith's contact information" target="_self">contact</a>          </p>
        <p>My research centers around the mechanisms that control the activities of voltage-gated proton channels (HV1) and NADPH oxidases (NOX), proteins that participate in cell signaling. Noxes and HV1 are known to work together in several cell types, and both produce signals important not only to human health but also to the development, physiology, and lifestyles of many other organisms.  The lab uses a combination of computational (bioinformatics, phylogenetics, and molecular structure) and laboratory (molecular biology, biochemistry, biophysics) methods to investigate how these proteins work.   In addition to what is going on right in the lab, we collaborate with labs doing drug discovery, basic and applied biomedical research, and basic and applied marine biological research - providing a rich context for a big picture view of what we do. <a href="http://ksuweb.kennesaw.edu/~ssmit325/" title="Dr. Smith's Research Information" target="_self"> Click for more information about her research</a>        </p>
        <p><strong><img class="imgr" src="../images/125x125/bio-filler_125X125.jpg" alt="Photo of Dr. Dale Lynn Vogelien" title="Photo of Dr. Dale Lynn Vogelien" width="125" height="125" /><a name="jump16" id="jump16"></a>Dale Lynn Vogelien, Ph. D.</strong><br />
        <a href="../people/dvogelien.html" title="Dr. Dale Lynn Vogelien's contact information" target="_self">contact</a><br />
      <p><strong>Research interests:&nbsp;</strong>My research interests lie in three distinct areas: plant  stress physiology, the development of a fingerprint analysis for daylily  varieties, and curriculum development.</p>
      <p>One ongoing project involves the using C-Fern to study the  mechanisms associated with cadmium (Cd) toxicity and tolerance and identify  genes involved. Heavy metal toxicity impacts a wide variety of plant  biochemical and physiological processes. Metal homeostasis and tolerance are  complex and involve regulating uptake and transport of metal ions, chelation of  metal ions upon entry into the cytoplasm and/or compartmentalization of metal  ions within cells and tissues. The most effective approach in sorting out how  the contributing mechanisms interact in a way that translates into an effective  strategy(s) for metal tolerance is to use mutants tolerant to higher levels of  a given metal in comparative studies with a sensitive wild type strain.  Students have successfully isolated three putative Cd-tolerant mutants of  C-Fern (M1 generation). Future studies will involve confirming the tolerance of  M2 spores to Cd, genetic characterization of the putative mutants, comparative  studies using mutant and wild type strains to examine Cd accumulation and ion  composition of gametophytes and sporophytes, and cross-tolerance studies  examining the tolerance of wild type and mutant strains to other heavy metals.</p>
      <p>A second area of interest is in the development of molecular  fingerprints for daylily varieties. An estimated 52,000 daylily varieties are  currently registered with the American Hemerocallis Society, with new varieties  added regularly. Currently, varieties are identified by defined morphological  descriptors, such as flower color and size, foliage features, etc.  Morphological characters, however, can often be influenced by growing  conditions, plant health and plant age, and thus can be insufficient or  unreliable for clear identification. Thus far, studies have focused on the use  of PCR-based approaches, including the use of random primers to generate RAPDS  (randomly amplified polymorphic DNA sequences) and AFLP (amplified fragment  length polymorphisms). These molecular markers would then be used to construct  a molecular fingerprint for a given variety.</p>
      <p>With respect to curriculum development, I am interested in  having students develop laboratory exercises for upper level biology and  biotechnology courses that use a current methodology/technique to study  interesting and relevant phenomena or problems. Two recently developed  laboratory exercises for a senior level plant physiology course include  'Quantification of ABA Levels in Plant Tissues Experiencing Increasing Levels  of Soil Desiccation Using an Immunoassay' and 'Analysis of Light Induced  Proteins in Pea Shoots using SDS-PAGE'. Both exercises were developed by  undergraduate students.</p>
      <p><br />
        <a href="#jumptotop" title="back to top">(Back to top)</a>      </p>
    </div>
    </div>
<!-- InstanceEndEditable -->
  <div id="column">
      <div class="holder">
        <h2 class="title"><a href="../index.html" target="_self"><img src="../images/icons/30/home.png" alt="Department of Biology and Physics home icon" width="30" height="30" title="Department of Biology and Physics home icon" border="0" /></a><a href="https://www.facebook.com/pages/Biology-Physics-at-Kennesaw-State-University/158317140833" target="_blank"><img src="../images/icons/30/fb.png" alt="Facebook icon" title="Facebook icon" width="30" height="30" border="0" /></a><a href="http://www.linkedin.com/groups/Kennesaw-State-University-College-Science-4705525" target="_blank"><img src="../images/icons/30/in.png" alt="Linkedin icon" width="30" height="30" border="0" title="Linkedin icon" /></a><a href="../people/faculty-staff.html" title="Directory icon" target="_self"><img src="../images/icons/30/directory.png" alt="Directory icon" title="Directory icon" width="30" height="30" border="0" /></a><a href="../about/contact.html" target="_self"><img src="../images/icons/30/email.png" alt="Contact Icon" title="Contact Icon" width="30" height="30" border="0" /></a><a href="../happening/calendar.html" target="_self"><img src="../images/icons/30/cal.png" alt="Calendar icon" title="Calendar icon" width="30" height="30" border="0" /></a></h2>
        <!-- InstanceBeginEditable name="column" -->
        <div id="featured">
          <ul>
            <li>
              <h2>Research at KSU</h2>
              <p class="imgholder"><a href="http://www.kennesaw.edu/research/"><img src="../images/240x90/feature-research.jpg" alt="Research at Kennesaw State University; Link to website" width="240" height="90" title="Research at Kennesaw State University; Link to website" border="0" align="middle" /></a></p>
              <strong><a href="http://www.kennesaw.edu/research/" title="Kennesaw State University Office of Research" target="_self">KSU Office of Research</a><br />
              <br />
              <a href="http://www.kennesaw.edu/research/#resources" title="KSU Research Resources" target="_self">KSU Research Resources</a></strong><br />
              <br />
              <a href="http://www.kennesaw.edu/ksursf/" title="The Kennesaw State University Research and Service Foundation (KSURSF)" target="_self"><strong>Kennesaw State University Research and Service Foundation (KSURSF)</strong></a><br />
              <br />
              <strong><a href="http://www.kennesaw.edu/research/proposals_awards.html" title="Kennesaw State University Proposals and Awards" target="_self">Proposals and Awards</a></strong><br />
              <br />
              <a href="http://cetl.kennesaw.edu/undergraduate-research-and-creative-activity" title="KSU Undergraduate Research" target="_self"><strong>KSU Undergraduate Research</strong></a><br />
              <br />
              <strong><a href="http://science.kennesaw.edu/resources/mentor.html" title="College of Science and Mathematics Mento Protege" target="_self">CSM Mentor Prot&eacute;g&eacute;</a></strong><br />
            </li>
          </ul>
        </div>
        <em>Page last updated: 
        <!-- #BeginDate format:Am1 -->October 31, 2013<!-- #EndDate -->
        </em>
        <!-- InstanceEndEditable --></div>
    </div>
    <div class="clear"></div>
  </div>
</div>
<!-- ####################################################################################################### -->
<div class="wrapper col4">
  <div id="footer">
    <div class="footbox">
      <h2><a href="../about/index.html" title="About" target="_self">About</a></h2>
      <ul>
        <li><a href="../about/contact.html" title="Contact" target="_self">Contact</a></li>
        <li><a href="../about/directions.html" title="Directions" target="_self">Directions</a></li>
        <li><a href="../about/socialmedia.html" title="History" target="_self">Social Media</a></li>
      </ul>
    </div>
    <div class="footbox">
      <h2><a href="../academics/index.html" title="Academics" target="_self">Academics</a></h2>
      <ul>
        <li><a href="../academics/certiticate/index.html" title="Certificate" target="_self">Certificate</a></li>
        <li><a href="../academics/undergraduate/index.html" title="Undergraduate" target="_self">Undergraduate</a></li>
        <li><a href="../academics/graduate/index.html" title="Graduate" target="_self">Graduate</a></li>
      </ul>
    </div>
    <div class="footbox">
      <h2><a href="../resources/index.html" title="Resources" target="_self">Resources</a></h2>
      <ul>
        <li><a href="../research/index.html" title="Research" target="_self">Research</a></li>
        <li><a href="../resources/compliance.html" title="Safety Compliance" target="_self">Safety Compliance</a></li>
        <li><a href="../resources/advising/index.html" title="Advising" target="_self">Advising</a></li>
        <li><a href="../resources/courses.html" title="Courses" target="_self">Courses</a></li>
        <li><a href="../faculty/index.html" title="Faculty and Staff Resources" target="_self">Resources - Faculty and Staff</a></li>
        <li><a href="../students/index.html" title="Student Resources" target="_self">Resources - Students</a></li>
      </ul>
    </div>
    <div class="footbox">
      <h2><a href="../people/index.html" title="People" target="_self">People</a></h2>
      <ul>
        <li><a href="../people/alumni.html" title="Alumni and Friends" target="_self">Alumni and friends</a></li>
        <li><a href="../people/faculty-staff.html" title="Faculty and Staff" target="_self">Faculty and Staff</a></li>
      </ul>
    </div>
    <div class="footbox last">
      <h2><a href="../happening/index.html" title="Happening" target="_self">Happening</a></h2>
      <ul>
        <li><a href="../happening/calendar.html" title="Calendar" target="_self">Calendar</a></li>
        <li><a href="../happening/events/index.html" title="Events" target="_self">Events</a></li>
        <li><a href="../news/index.html" title="News" target="_self">News</a></li>
      </ul>
    </div>
    <br class="clear" />
  </div>
</div>
<!-- ####################################################################################################### -->
<div class="wrapper col5">
  <div id="copyright">
    <div align="center"><a href="../index.html" title="Department of Biology and Physics" target="_self" class="active">Department of Biology and Physics</a><a href="http://science.kennesaw.edu" title="College of Science and Mathematics" target="_self" class="active">College of Science and Mathematics</a><a href="http://kennesaw.edu/" title="Kennesaw State University" target="_self">Kennesaw State University</a><a href="www.kennesaw.edu" title="Kennesaw State University" target="_self"><br />
      </a><a href="../site/index.html" title="About This Site" target="_self">About This Site</a><a href="../site/webmaster.html" title="Contact Webmaster" target="_self" class="active">Webmaster</a><a href="#" title="copyright 2011" target="_self">Copyright &copy; 2013</a>
      <div id="copyright2">
        <div align="center"></div>
      </div>
    </div>
  </div>
</div>
</body>
<!-- InstanceEnd --></html>
